$OpenBSD: patch-configure,v 1.3 2013/10/26 07:39:19 brad Exp $

Remove GNU make check since its broken with newer versions.

--- configure.orig	Sun Jan 27 11:51:06 2013
+++ configure	Sat Oct 26 00:04:35 2013
@@ -45,7 +45,6 @@ if tibs.TARGET [0] == "windows":
     }
 
 # Check for tools
-tibs.check_program ("GNU Make", "make --version", ".*?([0-9\.]+).*", "3.81", True)
 if tibs.check_program ("makedep", "makedep -V", ".*Version.*?([0-9\.]+).*", "0.1.0"):
     tibs.add_config_mak ("MAKEDEP", "makedep")
     makedep = "yes"
